h2o-docs/src/product/data-munging/target-encoding.rst

```diff
@@ -337,9 +338,13 @@ We do not need to apply any of the overfitting prevention techniques because our
 
    .. code-block:: python
 
-    ext_test = TargetEncoder(x=predictor, y=response,
-                             fold_column="fold", holdout_type="none", 
-                             blended_avg=False, noise=0, seed=1234)
+    # Add a fold column to the test set
+    fold = test.kfold_column(n_folds=5, seed=1234)
```

Comment:
@angela0xdata there is no need to have fold column when we want to apply target encoding to test data. You need to create encoding map based on training data(or holdout data that model are not going to be trained on) and then just apply it to test frame. Test frame should not have fold column at all.

Reply:
we just need to specify `fold_column` so that we can group encodings in our encoding map as they are split into folds in case when we generated map with folds for `kfold` application strategy.